src/main/metabase/examples.ts

```diff
@@ -191,8 +191,54 @@ function columnExampleQueries(field: Field, table: Table, aidColumns: string[]):
   }
 }
 
+function interactionSQL(fieldA: Field, fieldB: Field, table: Table): ExampleQuery {
+  const query: ExampleQuery = {
+    name: `${table.display_name} by ${fieldA.display_name} and ${fieldB.display_name}`,
```

Comment:
Maybe `Y by X`? E.g. `Gender by School`

Reply:
oh yeah, nice!

Reply:
or maybe not :joy: . The table cells don't contain `Gender` (per your example) values, but counts, so it's not strictly correct way to call it `Gender by something`